sass/give.scss

```diff
@@ -4,6 +4,14 @@
 @import "filter-styles";
 @import "breakpoints";
 
+.filter-collapse {
+  padding: 0.5rem 1rem;;
```

Comment:
I think it's a little confusing to have padding dependent solely on the order that the CSS is loaded here? I'd say either the CSS here should be more specific (nested under some `embed` specific class) or the `.filter-collapse { padding }` in `filter-styles.scss` should be moved to the main CSS since it's specific to that.

Also, you have two semicolons here.